src/uct/ib/rc/verbs/rc_verbs_impl.h

```diff
@@ -162,9 +162,9 @@ uct_rc_verbs_iface_fill_inl_am_sge_iov(uct_rc_verbs_iface_t *iface, uint8_t id,
 
 #define UCT_RC_VERBS_FILL_INL_PUT_WR(_iface, _raddr, _rkey, _buf, _len) \
     _iface->inl_rwrite_wr.wr.rdma.remote_addr = _raddr; \
-    _iface->inl_rwrite_wr.wr.rdma.rkey        = uct_ib_md_direct_rkey(_rkey); \
-    _iface->inl_sge[0].addr      = (uintptr_t)_buf; \
-    _iface->inl_sge[0].length    = _len;
+    _iface->inl_rwrite_wr.wr.rdma.rkey        = _rkey; \
```

Comment:
UCT_RC_VERBS_FILL_INL_PUT_WR is already called after converting uct_rkey_t to IB rkey